<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Team Teamâ„¢ Sprite Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="imageViewer">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>800</width>
      <height>800</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="addFrameBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>770</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Frame</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteFrameBtn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>770</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Frame</string>
    </property>
   </widget>
   <widget class="QListWidget" name="frameListView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>530</y>
      <width>241</width>
      <height>231</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <item>
     <property name="text">
      <string>Base Frame</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="mirrorBtn">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>650</y>
      <width>171</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string>Mirror Canvas</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="rectangleBtn">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>650</y>
      <width>171</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string>Rectangle Selection</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="clearCanvasBtn">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>460</y>
      <width>171</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Canvas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eraseBtn">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>460</y>
      <width>171</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string>Erase</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="ColorButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>161</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Click for color selection</string>
    </property>
   </widget>
   <widget class="QLabel" name="animatedFrame">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>70</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="animationSpeed">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>310</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="FullSize">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>30</y>
      <width>73</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Full Size</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Zoom">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>30</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Close Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveBtn_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadBtn_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="saveBtn">
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="loadBtn">
   <property name="text">
    <string>Load Project</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
